app/Http/Controllers/Backend/UserController.php

```diff
@@ -33,8 +33,8 @@ public function __construct(User $user)
      */
     public function index()
     {
-        $users = $this->user->all();
-        return view('backend.user.list',compact('users'));
+        $users = User::orderBy('id', 'ASC')->paginate();
```

Comment:
default orderBy is ASC, don't use orderBy('id', 'ASC')
if you want orderBy DESC, can use $users = User::orderBy('id', 'DESC')